cli/operator/node.go

```diff
@@ -539,7 +541,17 @@ func setupSSVNetwork(logger *zap.Logger) (networkconfig.NetworkConfig, error) {
 	if err != nil {
 		return networkconfig.NetworkConfig{}, err
 	}
-
+	if strings.HasPrefix(cfg.SSVOptions.CustomDomainType, "0x") {
```

Comment:
do we want to consider a case where its not starting with "0x"?

Reply:
@y0sher you're right we should err in this case rather than silently ignore